x-pack/plugins/observability_solution/synthetics/server/queries/query_monitor_status.ts

```diff
@@ -201,7 +201,7 @@ export async function queryMonitorStatus(
         configId: `${monitorQueryIdToConfigIdMap[queryId]}`,
         monitorQueryId: queryId,
         status: 'unknown',
-        location: loc,
+        locationId: loc,
```

Comment:
Unrelated to settings.

Reply:
i think this better reflects the variable. since it's a string.